tests/Microsoft.Bot.Builder.LUIS.Tests/LuisRecognizerMiddlewareTests.cs

```diff
@@ -0,0 +1,67 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.Bot.Builder.Adapters;
+using Microsoft.Bot.Builder.Core.Extensions.Tests;
+using Microsoft.Cognitive.LUIS;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Microsoft.Bot.Builder.LUIS.Tests
+{
+    [TestClass]
+    public class LuisRecognizerMiddlewareTests
+    {
+        private readonly string _luisAppId = TestUtilities.GetKey("LUISAPPID");
+        private readonly string _subscriptionKey = TestUtilities.GetKey("LUISAPPKEY");
+        private readonly string _luisUriBase = TestUtilities.GetKey("LUISURIBASE");
+
+        [TestMethod]
+        public async Task LuisRecognizer_TestMiddleware()
+        {
+            var adapter = new TestAdapter()
```

Comment:
Consider adding logic to NOT run these tests if the keys are not present. This is something we need to do more broadly anyway...

Reply:
Done